app/models/validations/setup_validations.rb

```diff
@@ -5,6 +5,33 @@ def validate_irproduct_other(record)
     end
   end
 
+  def validate_startdate(record)
+    if record.startdate > Time.zone.today
+      record.errors.add :startdate, I18n.t("validations.setup.startdate.today_or_earlier")
+    end 
+    
+    if record.voiddate.present?
+      if (record.startdate.to_date - record.voiddate.to_date).to_i.abs > 730
+        record.errors.add :startdate, I18n.t("validations.setup.startdate.voiddate_difference")
+      end 
+    end 
+
+    if record.mrcdate.present?
```

Comment:
Is this not already covered by https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/blob/main/app/models/validations/date_validations.rb#L12?